force-app/main/default/classes/generateCertificates.cls

```diff
@@ -0,0 +1,43 @@
+public class generateCertificates {    
```

Comment:
A class should start with the capital letter generateCertificates -> **G**enerateCertificates. I think something like "CertificateGenerator" is a better name here.